Dockerfile

```diff
@@ -1,41 +1,37 @@
-FROM golang:1-alpine
+FROM golang:1
 
 ENV PROJECT=coco-synthetic-image-publication-monitor
-COPY . /${PROJECT}-sources/
+ENV BUILDINFO_PACKAGE="github.com/Financial-Times/service-status-go/buildinfo."
+
 COPY kubeconfig-template /root/.kube/config
 COPY start.sh /
 
+COPY . /${PROJECT}/
+WORKDIR /${PROJECT}
 
 RUN wget "https://storage.googleapis.com/kubernetes-release/release/v1.13.7/bin/linux/amd64/kubectl" -O /usr/local/bin/kubectl \
   && chmod +x /usr/local/bin/kubectl
```

Comment:
This is missing in the scratch image. As far as I can see it is really necessary for the service to work.